NEWS.md

```diff
@@ -2,15 +2,18 @@
 
 ## Breaking Changes
 
+- Phi, Cohen's *w*, Cramer's *V*, ANOVA effect sizes, rank Epsilon squared, Kendall's *W* - CIs default to 95% one-sided CIs (`alternative = "greater"`).
```

Comment:
Maybe also mention here what the users can do to reproduce results from previous version of `effectsize`.